HW_10/task_1.ts

```diff
@@ -0,0 +1,70 @@
+function getFirstElement<T>(arr: T[]): T {
+    return arr[0];
+}
+
+console.log(getFirstElement<string>(['1', '2']))
+console.log(getFirstElement<number>([1, 2]))
+
+interface IPerson {
+    name: string;
+    surname: string;
+    experienceOfYears: number;
+    getDetails(): string;
+}
+
+abstract class Employee implements IPerson {
+    protected salary = 0;
+    constructor(
+        public name: string,
+        public surname: string,
+        public experienceOfYears: number
+    ) {};

```

Comment:
Add there calculateSalary method and you won't be needed to add it the all child classes